### PR TITLE
git-tag-project-version-and-build-number 1.0.0

### PR DESCRIPTION
**Pull request by ned1988**
### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

File: steps/git-tag-project-version-and-build-number/1.0.0/step.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    summary: Path to the 'Info.plist' file
+    title: Path to the 'Info.plist'
+- bitrise_tag_format: v%s(%s)
+  opts:
+    description: |
+      Provide format of generated tag & build number
+    is_required: false
+    summary: Provide format of generated tag & build number
+    title: Result format
+outputs:
```
Comment (trapacska) [inline steps/git-tag-project-version-and-build-number/1.0.0/step.yml]: please remove the whole outputs block if the step don't uses them

**Comment by ned1988** [comment]
@trapacska Done 👍 Ready for round 2

**Comment by koral--** [comment]
Since there is no output variables generated by step you should also remove `envman add --key EXAMPLE_STEP_OUTPUT --value 'the value you want to share'` from here https://github.com/ned1988/bitrise-step-git-tag-project-version-and-build-number/blob/master/step.sh#L73 as well as envstore file.

**Comment by ned1988** [comment]
@koral-- Done 👍

**Comment by trapacska** [comment]
Thx @ned1988 @koral-- @BirmacherAkos !

🦄

**Comment by BirmacherAkos** [comment]
Hi!

There was an issue with releasing the step.
Error: Step commit hash (e2cb1a7eff22ceca25843e0eaa65adf489e9dc1a) should be the  latest commit hash (7d6ce6e2e1d0d03db147e296ea86d863731655c1) on git tag.

This could happen when you manually change the git tag after you called the 
`bitrise create-release` workflow.
You should not do that.

The solution is:
You need to create a new release with v1.0.1.

**Comment by ned1988** [comment]
Whoops! 😯 Sorry for that, will try to fix asap.

**Comment by ned1988** [comment]
@BirmacherAkos created new [PR](https://github.com/bitrise-io/bitrise-steplib/pull/1567)

**Pull request merged**